Tasks/Common/AzureRmDeploy-common/operations/AzureAppServiceUtility.ts

```diff
@@ -217,6 +224,10 @@ export class AzureAppServiceUtility {
         }
     }
 
+    private startupCommandRecommendation(runtimeStack: string) {
+        // Add startup command recommendations for different runime environments
```

Comment:
Is this a TODO and beyond the scope of the PR ?

Reply:
Yes. Its a place holder to add some recommendations as followup of this PR

Reply:
Let's add this when required.